ios/Classes/BetterPlayerPlugin.m

```diff
@@ -417,6 +417,10 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
             } else {
                 [self disposeNotificationData:player];
             }
+            
+            BOOL isLiveStream = [self isLiveStream:player];
+            [player setIsHidedPipSeekButtons:isLiveStream];
+            player._isLiveStream = isLiveStream;
```

Comment:
We use `_` ofter for property only use in parent class, please make a set function for it.